### PR TITLE
Add readme

### PR DESCRIPTION
**Pull request by brinxmat**
Adds scope for project

### CODE REVIEW

**Review by torarnet — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    -   DAOs (TODO)
+    -   JSON-LD contexts for the DTOs (TODO)
+
+## Working with the data model
+
+The main point-of-reference for users of the data model is the DTOs; the client
+APIs simplify working with these.
+
+The clients can be used via maven/gradle (Java) and npm (JavaScript):
+-   gradle: ```shell TODO```
```
Comment (mattiahj) [inline README.md:27]: I'll approve, but did you mean to finish these TODOs in this PR?

**Review by mattiahj — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
+# NVA data model
+
+This is version 2 of the data model, as it entirely differs from the approach 
+taken in [version 1](https://github.com/BIBSYSDEV/nva-datamodel-java), the 
+decision was taken to separate the code entirely.
+
+The data model comprises the following components:
+-   Data Transfer Objects (DTOs) (TODO)
+
+-   Data Access Objects (DAOs) (TODO)
```
Comment (axthosarouris) [inline README.md:10]: Should the DAO model be internal in the publication service?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    -   JSON-LD contexts for the DTOs (TODO)
+
+## Working with the data model
+
+The main point-of-reference for users of the data model is the DTOs; the client
+APIs simplify working with these.
+
+The clients can be used via maven/gradle (Java) and npm (JavaScript):
+-   gradle: ```shell TODO```
+-   maven: ```shell TODO```
```
Comment (axthosarouris) [inline README.md:28]: I think maven support should be left until necessary.

File: README.md (hunk 1 above)
Comment (brinxmat) [reply README.md:27]: No, they need doing at a point when I know what I am going to write here :)

File: README.md (hunk 3 above)
Comment (brinxmat) [reply README.md:28]: As it is the client, this is a low priority, agreed, but we have several integrators that use Maven

**Review by janbj — APPROVED** [review]

**Pull request merged**